com.unity.shadergraph/Documentation~/Vector-1-Node.md

```diff
@@ -1,20 +1,20 @@
-# Vector 1 Node
```

Comment:
I'd like to know whether we should rename this page to `Float.md` (as well as update all links to `Vector-1-Node.md`).

Reply:
hmm... good point. We should rename it. Will you do it @sharlenet or you expect I do ?

Reply:
I've made the changes and will approve this PR.